src/components/projects/lean-workflow/CommentList.tsx

```diff
@@ -121,11 +131,68 @@ export const CommentList: React.FC<Props> = ({ selected, editor }) => {
       if (highlight) {
         // newComment.originalText = getHighlightText(highlight, state)
         newComment.originalText = highlight.text
-        setNewComment(newComment)
       }
+      setNewComment(newComment)
     }
   }, [commentTarget, doc, newComment, state, currentUser])
 
+  const commentsLabel = useMemo(() => {
```

Comment:
please add some comments to this function explaining what is going on inside so it will be easier to work with it in future, thanks

Reply:
Also, if I understand correctly, this function collects all the labels from the doc so perhaps it should be called commentsLabels or getCommentsLabels?

Reply:
added a comment to clarify it more, and update the name to `commentsLabels`